tests/ctst/features/quotas/CountItems.feature

```diff
@@ -0,0 +1,12 @@
+Feature: CountItems measures the utilization metrics
+  The utilization metrics are computed for accounts, buckets and locations
+
+@2.6.0
+@PreMerge
+@CronJob
+@CountItems
+Scenario Outline: Countitems runs without error and compute utilization metrics
+    Given an existing bucket "" "without" versioning, "without" ObjectLock "" retention mode
+    And an object "" that "exists"
+    When the CountItems cronjobs completes without error
```

Comment:
can "CountItems" be parameterized ? to be reused for another job if we want ?

Reply:
If we have only 1 cronjob in the step, it's the same as having one implementation for each (and eventually call a generic function), but let's parameterize it